Description: Use if let instead of matches to maintain MSRV 1.41.1
 .
 electrs (0.9.3-1) buster; urgency=high
 .
Author: Martin Habovstiak <martin.habovstiak@gmail.com>
Index: electrs-0.9.3/src/p2p.rs
===================================================================
--- electrs-0.9.3.orig/src/p2p.rs
+++ electrs-0.9.3/src/p2p.rs
@@ -244,7 +244,7 @@ impl Connection {
                         }
                         NetworkMessage::Inv(inventory) => {
                             debug!("peer inventory: {:?}", inventory);
-                            if inventory.iter().any(|inv| matches!(inv, Inventory::Block(_))) {
+                            if inventory.iter().any(|inv| if let Inventory::Block(_) = inv { true } else { false }) {
                                 let _ = new_block_send.try_send(()); // best-effort notification
                             }
 
